### PR TITLE
Update the `core-tutorial` branch - Part 11

### PR DESCRIPTION
**Pull request by dhmlau**
Update the `core-tutorial` branch - part 11 (summary)

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/tutorials/core/11-summary.md [hunk 1]
```diff
@@ -1,142 +1,59 @@
...
+permalink: /doc/en/lb4/core-tutorial-part11.html
+---
+
+In the previous sections, we have explained how various features in LoopBack
+core modules can be useful for our developers to build a scalable Node.js
+application.
 
-https://12factor.net/
+LoopBack can also be used as the framework to create microservices. The
+[Twelve-Factor App](https://12factor.net) methodology is a methodology for
```
Comment (raymondfeng) [inline docs/site/tutorials/core/11-summary.md]: methodology promotes some key architectural principles for

File: docs/site/tutorials/core/11-summary.md [hunk 2]
```diff
@@ -1,142 +1,59 @@
...
+building software-as-a-service applications. This section shows how the criteria
+of Twlve-Factor are satisfied by Node.js application powered by LoopBack.
```
Comment (raymondfeng) [inline docs/site/tutorials/core/11-summary.md]: Twelve

File: docs/site/tutorials/core/11-summary.md [hunk 3]
```diff
@@ -1,142 +1,59 @@
...
-> VII. Port binding Export services via port binding
+### VII. Port binding - Export services via port binding
 
 HTTP/HTTPS server via configuration
 
-> VIII. Concurrency Scale out via the process model
+### VIII. Concurrency - Scale out via the process model
 
-> IX. Disposability Maximize robustness with fast startup and graceful shutdown
+### IX. Disposability - Maximize robustness with fast startup and graceful shutdown
```
Comment (raymondfeng) [inline docs/site/tutorials/core/11-summary.md]: We probably should clean up this page to avoid 1-1 mapping between LB4 and 12 factors.

**Comment by dhmlau** [comment]
@raymondfeng, I've reworked the summary according to your feedback. PTAL again. Thanks.

**Pull request merged**